docs/runbook/setting-up-circle-ci.md

```diff
@@ -0,0 +1,25 @@
+# Setting up CircleCI
+
+The application is configured to run automated tests via CircleCI. The configuration can be found in [circle.yml](../../circle.yml). Once set up, CircleCI will ensure that changes in Pull Requests pass our test suite before they are allowed to be merged.
+
+CircleCI can be enabled for your repository by setting up a new plan (https://github.com/marketplace/circleci). This will add CircleCI as a [status check](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) to your repo.
+
+Once you have set up a new CircleCI plan, enable the use of 3rd party orbs (https://circleci.com/docs/2.0/orbs-faq/#using-3rd-party-orbs).
+
+Finally, set the required environment variables for the project (https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project).
```

Comment:
```suggestion
Once you have set up a new CircleCI plan:

1. Enable the use of 3rd party orbs (https://circleci.com/docs/2.0/orbs-faq/#using-3rd-party-orbs).
2. Set the required environment variables for the project (https://circleci.com/docs/2.0/env-vars/#setting-an-environment-variable-in-a-project).
```

Maybe? I find using an ordered list when providing instructions makes it clearer that these are steps that the person will need to take.